pkg/loader/compose/compose.go

```diff
@@ -207,6 +207,15 @@ func (c *Compose) LoadFile(file string) kobject.KomposeObject {
 				}
 			}
 
+			// canonical custom label handler
```

Comment:
mind adding another comment as to what this actually does, other than "custom label handler"? example?